packages/front-end/components/PowerCalculation/stats.ts

```diff
@@ -0,0 +1,354 @@
+import normal from "@stdlib/stats/base/dists/normal";
+
+import {
+  PowerCalculationParams,
+  PowerCalculationResults,
+  MDEResults,
+  SampleSizeAndRuntime,
+  Week,
+} from "./types";
+
+/**
+ * delta method for relative difference
+ *
+ * @param varA Scalar control distribution variance.
+ * @param meanA Scalar control mean.
+ * @param nA Control sample size.
+ * @param varB Scalar treatment distribution variance.
+ * @param meanB Scalar treatment distribution mean.
+ * @param nB Treatment sample size.
+ * @param relative boolean indicator for relative effects.
+ * @returns variance.
+ */
+export function frequentistVariance(
+  varA: number,
+  meanA: number,
+  nA: number,
+  varB: number,
+  meanB: number,
+  nB: number,
+  relative: boolean
+): number {
+  if (relative) {
+    return (
+      varB / (Math.pow(meanA, 2) * nB) +
+      (varA * Math.pow(meanB, 2)) / (Math.pow(meanA, 4) * nA)
+    );
+  } else {
+    return varB / nB + varA / nA;
+  }
+}
+
+export function powerStandardError(
+  variance: number,
+  mean: number,
+  n: number,
+  nVariations: number,
+  effectSize: number
+): number {
+  return Math.sqrt(
+    frequentistVariance(
+      variance,
+      mean,
+      n / nVariations,
```

Comment:
nit: I think this code could probably be agnostic to `nVariations` (i.e. it should happen upstream) so that later if we wanted something like different weights per variations or what not it wouldn't be happening within `powerStandardError` which probably doesn't need to be aware of that.

Reply:
updated to remove number of variations